styles.css

```diff
@@ -0,0 +1,90 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+html {
+  box-sizing: border-box;
+  height: 100%;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+.heading-section {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: space-around;
+  padding: 30px 20px;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  margin: 0;
+  justify-content: space-between;
+  color: white;
+  background-color: #3c3a39;
+  text-decoration: none;
+  padding: 20px;
+}
+
+body {
+  background-color: #1c1a19;
+  margin: 0;
+  height: 100%;
+  width: 100%;
+  background-image: url(images/Group.png);
+  background-repeat: no-repeat;
+  background-position-x: right;
+  background-position-y: bottom;
+}
```

Comment:
- [ ] [OPTIONAL] Kindly make sure your `background-image` is aligned properly as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/205052265-00115aed-6b41-459d-9f0b-823d5a5870ae.png)  |  ![image](https://user-images.githubusercontent.com/74814780/205052094-7a55f672-1804-4c64-9553-a41be74109e3.png)